mentoring/mentoring.py

```diff
@@ -348,6 +352,7 @@ def handleAssessmentSubmit(self, submissions, suffix):
                     'score_type': 'proficiency',
                 })
                 event_data['final_grade'] = score.raw
+                message = self.get_message_html('on-assessment-review')
```

Comment:
Can't you just assign `self.assessment_message` here instead of calling `get_message_html`?